src/Conf.ml

```diff
@@ -789,6 +772,70 @@ let default =
   ; wrap_comments= C.default Formatting.wrap_comments
   ; wrap_fun_args= C.default Formatting.wrap_fun_args }
 
+let janestreet_profile =
+  { default with
+    break_cases= `Fit
+  ; break_infix= `Fit_or_vertical
+  ; break_string_literals= `Wrap
+  ; doc_comments= `Before
+  ; extension_sugar= `Preserve
+  ; field_space= `Loose
+  ; if_then_else= `Keyword_first
+  ; indicate_nested_or_patterns= false
+  ; infix_precedence= `Parens
+  ; leading_nested_match_parens= true
+  ; let_and= `Sparse
+  ; let_open= `Preserve
+  ; margin= 90
+  ; ocp_indent_compat= true
+  ; parens_tuple= `Multi_line_only
+  ; sequence_style= `Terminator
+  ; type_decl= `Sparse
+  ; wrap_fun_args= false
+  ; module_item_spacing= `Compact }
+
+let preset_profiles =
+  [ ( "janestreet"
+    , `JaneStreet
```

Comment:
Is this variant really useful ?

Reply:
I suppose that we will soon also have 'compact' and 'sparse' profiles, at which point the variant may be more useful.

Reply:
And of course there should also be a 'default' profile.